### PR TITLE
resolve bugs of 3.4.0-preview

### PR DESCRIPTION
**Pull request by ramoka178**

### CODE REVIEW

File: eng/mgmt/mgmtmetadata/resources_resource-manager.txt [hunk 1]
```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --tag=package-resources-2019-10 --csharp.output-folder=Y:\generated\10-01
-2020-05-12 04:45:33 UTC
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --tag=package-subscriptions-2020-01 --csharp.output-folder=D:\workspace\azure-sdk-for-net\sdk\resources\Microsoft.Azure.Management.Resource\src\Generated\Subscriptions
```
Comment (hamidtirmizi) [inline eng/mgmt/mgmtmetadata/resources_resource-manager.txt:6]: I don't know how this part works, but is that a path local to your machine, which may not work?
Also why the change from `resources` package to `subscriptions` package?

Comment (ramoka178) [reply eng/mgmt/mgmtmetadata/resources_resource-manager.txt:6]: I believe, this is more of for audition purposes, to see what had changed.
Resources 2019-10 package was also run including the Subscription 2020-01 package, looks like only the change is being added here. 

@isra-fel : could you comment if there is any issue with the changes here ?

**Review by hamidtirmizi — APPROVED** [review]

**Review by isra-fel — APPROVED** [review]
LGTM
